### PR TITLE
Add loading state for Menu and Content

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Add loading state for Menu and Content 

https://github.com/HedvigInsurance/racoon/assets/6661511/c1c88cf7-c634-4873-831f-d3f02f1da61d

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3830?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #3828** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3828?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3829** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3829?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3830** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3830?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3830?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]
Looks good!

File: apps/store/src/features/memberArea/components/Menu.tsx [hunk 1]
```diff
@@ -142,3 +146,29 @@ const NavigationButton = styled.button(NavigationItemStyles, {
...
+        <NavItemSkeleton />
+        <NavItemSkeleton />
+        <NavItemSkeleton />
+      </NavigationList>
+    </Navigation>
+  )
+}
+
+const NavItemSkeleton = styled(Skeleton)({
+  width: '165px',
```
Comment (guilhermespopolin) [inline apps/store/src/features/memberArea/components/Menu.tsx:166]: Should we convert those pixel values into `rem`s? 🤔

File: apps/store/src/features/memberArea/components/MemberAreaLayout.tsx [hunk 2]
```diff
@@ -39,8 +41,12 @@ export const MemberAreaLayout = ({ children }: Props) => {
         {headerBlock.map((nestedBlock) => (
           <HeaderBlock key={nestedBlock._uid} blok={nestedBlock} />
         ))}
-        {loading && <div>Loading...</div>}
-        {!loading && (
+        {loading ? (
```
Comment (guilhermespopolin) [inline apps/store/src/features/memberArea/components/MemberAreaLayout.tsx:44]: nit:
```
<Main>
  {loading ? (
    <MenuLoadingState />
    <ContentLoadingState />
  ) : (
    <Menu />
    <Content>{children}</Content>
  )}
</Main>
```

**Review by guilhermespopolin — APPROVED** [review]
Good job. Looking sharp :)
Just left some comments.

**Comment by gustaveen** [comment]
### Merge activity

* **Jan 11, 7:50 AM**: @gustaveen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3830).

**Pull request merged**